serves/healthcheck.go

```diff
@@ -0,0 +1,13 @@
+package serves
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func (a Serve) HealthCheck(w http.ResponseWriter, req *http.Request) {
```

Comment:
Hi @alidorludovic, looks like to me that the function sends "ok" without checking anything. Is that the idea behind what you want to do ?

Reply:
Hi @adamspd 

Thank you for your feedback!  

Indeed, the purpose of the /health route is to perform a simple liveness check to ensure the availability of the application. The response "ok" indicates that the application is up and running.

In addition, I've updated the nomenclature from /health to /healthy for better clarity and consistency. This change aligns with the intention of the route, emphasizing the positive state of the application.

If you have any further questions or suggestions, please feel free to let me know.